cdc/sink/codec/avro.go

```diff
@@ -542,13 +542,30 @@ func newAvroEventBatchEncoderBuilder(credential *security.Credential, opts map[s
 		return nil, cerror.ErrPrepareAvroFailed.GenWithStack(`Avro protocol requires parameter "registry"`)
 	}
 
+	topic, ok := opts["topic"]
+	if !ok {
+		topic = ""
+	}
+
+	nameStrategy, ok := opts["nameStrategy"]
+	if !ok {
```

Comment:
how about just directly returning the error, since the first two cases won't be hit I think.

Reply:
Fixed in 15290e028456b3e0c00c413e1e135cb3d81604c2